src/Finder/SourceFilesFinder.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Copyright © 2017 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+declare(strict_types=1);
+
+namespace Infection\Finder;
+
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @author Lander Vanderstraeten <lander_vanderstraeten@hotmail.com>
```

Comment:
@borNfreee do we really want to bother with the `@author` tags? In my experience the commit history is more than enough for this, those tags are just noisy and annoying to manage